axum/src/docs/method_routing/route_layer.md

```diff
@@ -2,9 +2,12 @@ Apply a [`tower::Layer`] to the router that will only run if the request matches
 a route.
 
 Note that the middleware is only applied to existing routes. So you have to
-first add your routes (and / or fallback) and then call `route_layer`
-afterwards. Additional routes added after `route_layer` is called will not have
-the middleware added.
+first add your routes and then call `route_layer` afterwards. Additional routes
+added after `route_layer` is called will not have the middleware added.
+
+**Important**: The middleware is NOT applied to fallback handlers. Fallback handlers
```

Comment:
This paragraph is redundant with the one after it. Maybe that one isn't quite clear, but I don't think leaving it as-is and adding an extra one before it is the right fix. Can you think about how to merge these two paragraphs?